lib/angular/sync-ng.js

```diff
@@ -21,8 +21,8 @@ angular.module('wfm.sync.service', [])
     });
   }
 
-  syncService.init = function($fh, syncOptions) {
-    sync.init($fh, syncOptions);
+  syncService.init = function(syncOptions) {
+    sync.init(window.$fh, syncOptions, mediator);
```

Comment:
Yeh, I agree. Ideally there would be an angular feedhenry sdk exposed by the application.

I'm also not sure how it was working in the application. It's very odd.